odl/tomo/analytic/filtered_back_projection.py

```diff
@@ -79,16 +80,16 @@ def _fbp_filter(norm_freq, filter_type, frequency_scaling):
     """
     if callable(filter_type):
         filt = filter_type(norm_freq)
-    elif filter_type == 'Ram-Lak':
+    elif filter_type.upper() == 'RAM-LAK':
```

Comment:
prefer `lower`. Uppercase is usually reserved for global constants and could be confusing.

Reply:
Ok. I thought about these strings more like global constants (or an enumeration type). But you're the boss. :smiley:

Reply:
ODL is a collaborative effort, I try avoiding being a boss the best I can :)

Anyway, I guess the above explanation was not perfect honestly, but I've always seen people use `str.lower()` to normalize capitalization, I tried googling it but didn't see any official guideline on it so perhaps it is just convention.